public/index.js

```diff
@@ -1,11 +1,19 @@
 var search_box = document.querySelector('#nav_search_box');
-search_box.addEventListener('input', search);
+search_box.addEventListener('change', search);
 function search(event){
-    searches = require('../exampleQuiz.json');
-    for(var i=0;i<searches.length;i++){
-        if((searches[i].name.includes(search_box.value)||(searches[i].description.includes(search_box.value))){
-            
-        }
-    }
+   // searches = require('exampleQuiz.json');
+    //for(var i=0;i<searches.length;i++){
+     //   console.log(searches)
+       // if((searches[i].name.includes(search_box.value))||(searches[i].description.includes(search_box.value))){
+            //FIX ME//
+            var url='/search/'+search_box.value.toLowerCase();
```

Comment:
We're going to need to replace spaces with something else. Could you look into that?

Reply:
Done

Reply:
Now it just removes spaces, but we can change that later if we need